docker/tensorrt/trt.hcl

```diff
@@ -0,0 +1,25 @@
+target deps {
+  dockerfile = "docker/main/Dockerfile"
```

Comment:
Each of these "imported" target definitions needs to specify the target to build, e.g. `target = "deps"`, otherwise they each trigger a full build of the entire dockerfile